### ISSUE TITLE
Not showing images on Android 14

### ISSUE THREAD

**Issue by slopsjon** [gotify/android#343]
`#!/bin/bash
TITLE="Motion"
MESSAGE="![](https://192.168.1.176:1880/$1)"
PRIORITY=10
URL="https://server.com/message?token=TOKEN!!!"

curl -s -S --data '{"message": "'"${MESSAGE}"'", "title": "'"${TITLE}"'", "priority":'"${PRIORITY}"', "extras": {"client::display": {"contentType": "text/markdown"}}}' -H 'Content-Type: application/json' "$URL"`

I had this working on my old phone, but I have switched from an S9 to a Galaxy S21 with Android 14 on it and now the images don't show, but I receive the notification. I have gone through all the settings on the phone to see if the app has the permissions and can't see what is blocking the images. Does anyone have this working on android 14. It still works on my old phone.

**Comment by jmattheis** [comment]
Can you open the image inside a webbrowser on your phone? Where are the images host, are they behind a cloudflare proxy?

**Comment by slopsjon** [comment]
Yes I can open the image using the url https://192.168.1.176:1880/image.png. The image is being served by my node red server. I have an nginx proxy, but I'm accessing the image with the ip address.

**Comment by jmattheis** [comment]
Can you check the logs of gotify/android? You are probably using a self-signed certificate which gotify/android isn't allowing.

**Comment by slopsjon** [comment]
I got this working now. 

I had to remove the cert files from node red settings file and then modify the nginx config for node red and then post the image as http, (as I've removed the cert from node red). So I can access node red and the dashboard with a self signed, but Gotify has to access the image as HTTP. 

Don't know why this only effected Gotify as I could access the node red server and dashboard from the phone, just not via Gotify, must be something changed in Android 14.
